source/fundamentals/data-formats/documents.txt

```diff
@@ -284,6 +289,94 @@ For more information on retrieving and manipulating MongoDB data, see our
+and passing it a
+:java-docs:`JsonWriterSettings</apidocs/bson/org/bson/json/JsonWriterSettings.html>`
+object. For more information on JSON formats, see
+our :doc:`Extended JSON guide </fundamentals/data-formats/document-data-format-extended-json/>`.
+For more information on specifying codecs, see our 
+:doc:`Codecs guide </fundamentals/data-formats/codecs>`.
+
+In the following code snippet, we show how to instantiate a sample ``JsonObject`` 
+instance wrapping an Extended JSON string containing different types of key value pairs:
+
+.. code-block:: java
+
+   String ejsonStr =  "{"
+                     +" \"_id\": {"
+                     +"  \"$oid\": \"6035210f35bd203721c3eab8\""
+                     +" },"
+                     +" \"name\": \"Gabriel Garc\\u00eda M\\u00e1rquez\","
+                     +" \"dateOfDeath\": {"
+                     +"  \"$date\": \"2014-04-17T04:00:00Z\""
+                     +" },"
+                     +" \"novels\": ["
+                     +"  {"
+                     +"   \"title\": \"One Hundred Years of Solitude\","
+                     +"   \"yearPublished\": 1967"
+                     +"  },"
+                     +"  {"
+                     +"   \"title\": \"Chronicle of a Death Foretold\","
+                     +"   \"yearPublished\": 1981"
+                     +"  },"
+                     +"  {"
+                     +"   \"title\": \"Love in the Time of Cholera\","
+                     +"   \"yearPublished\": 1985"
+                     +"  }"
+                     +" ]"
+                     +"}";
+   JsonObject author = new JsonObject(ejsonStr);
+
+To insert this document into a collection, instantiate a collection using
+:java-docs:`getCollection()</apidocs/mongodb-driver-sync/com/mongodb/client/MongoDatabase.html#getCollection(java.lang.String,java.lang.Class)>`,
+specifying the ``JsonObject`` class. Then, call the
+:doc:`insertOne </usage-examples/insertOne>` operation as follows:
+
+.. code-block:: java
+
+   // MongoClient mongoClient = <code to instantiate your client>;
+
+   MongoDatabase database = mongoClient.getDatabase("fundamentals_data");
+   MongoCollection<JsonObject> collection = database.getCollection("authors", JsonObject.class);
+
+   InsertOneResult result = collection.insertOne(author);
+
+Once you perform a successful insert, you can retrieve the sample JSON data from the 
+collection using the following code:
+
+.. code-block:: java
+
+   // MongoClient mongoClient = <code to instantiate your client>;
+
+   JsonObject query = new JsonObject("{\"name\": \"Gabriel Garc\\u00eda M\\u00e1rquez\"}");
+   JsonObject json = collection.find(query).first();
+   if (json != null) {
+       //this is equivalent to System.out.println("raw extended json string: " + json);
+       System.out.println("raw extended json string: " + json.getJson());
+   }
+
+The output should look something like this:
+
+.. code-block:: java
```

Comment:
This is output and not java code, so we shouldn't specify the java pygments highlighter as it might look random depending on what keywords it picks up. I think "none" works here since it's a blend of text and JSON.

Reply:
Thanks for the feedback. Apologies, I will give output the none type going forward. Change is here: https://github.com/mongodb/docs-java/pull/57/commits/26577c6da0c2e554f62a4410286f763c921ae80d

takeaway:

output gets code-block type `none`